client/src/templates/Challenges/redux/create-question-epic.js

```diff
@@ -83,14 +88,22 @@ function createQuestionEpic(action$, state$, { window }) {
         )}\n${i18next.t('forum-help.add-code-three')}\n\n\`\`\`\n${endingText}`
       );
 
+      const userText = user.name || user.username || '';
```

Comment:
After further thought, I do have some concerns about appending the user's name or username. Just because we aren't necessarily respecting their privacy settings.

I wonder if it would be better to use a hashing library to generate a random string with a low collision rate for the ends of the title?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
That should be ideal, yeah.

Reply:
This is implemented now in the latest commit.